inference-engine/tests/functional/plugin/shared/include/behavior/plugin/core_integration.hpp

```diff
@@ -930,6 +931,12 @@ TEST_P(IEClassNetworkTestP, LoadNetworkCreateDefaultExecGraphResult) {
     }
 }
 
+TEST_P(IEClassLoadNetworkTestWithThrow, LoadNetworkActualWithThrow) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    InferenceEngine::Core  ie = BehaviorTestsUtils::createIECoreWithTemplate();
+    ASSERT_THROW(ie.LoadNetwork(actualCnnNetwork, deviceName), InferenceEngine::Exception);
```

Comment:
not sure I understand: if the device name is "" lie in this test instance, the Core should replace that to the AUTO, isn't? why we throw then?

Reply:
As we discussed in meeting, AUTO will not support "" as device name after we enable AUTO as default device name. 

We remove the code to support "" device name in another PR. 

This PR will add test case to cover "" device name which will throw expectation as other unsupported device name.